compiler/shared_ast/expr.ml

```diff
@@ -931,3 +959,33 @@ let make_tuple el m0 =
         (List.map (fun e -> Marked.get_mark e) el)
     in
     etuple el m
+
+let translate_op_kind : type a. a op_kind -> 'b op_kind = function
+  | KInt -> KInt
+  | KRat -> KRat
+  | KMoney -> KMoney
+  | KDate -> KDate
+  | KDuration -> KDuration
+
+let translate_op : type a. a operator -> 'b operator = function
```

Comment:
wouahou that's a weird functions but yeah I get why you need it (GADTs...)

Reply:
Hm yeah :roll_eyes: it should be removed soon actually.